ara/cli/task.py

```diff
@@ -36,13 +36,37 @@ class TaskList(Lister):
     """Returns a list of tasks"""
     log = logging.getLogger(__name__)
 
-    def take_action(self, parsed_args):
+    def get_parser(self, prog_name):
+        parser = super(TaskList, self).get_parser(prog_name)
+        g = parser.add_mutually_exclusive_group(required=True)
+        g.add_argument(
+            '--playbook', '-b',
+            metavar='<playbook-id>',
+            help='Playbook from which to list tasks',)
+        g.add_argument(
+            '--play', '-p',
+            metavar='<play-id>',
+            help='Play from which to list tasks',)
+        g.add_argument(
+            '--all', '-a',
+            action='store_true',
```

Comment:
This should probably be the default behavior (or not require an argument at all), from looking at the code (haven't tested it yet) it doesn't look like this is the case